### DISCUSSION TITLE
VM lost access to the network

### DISCUSSION THREAD

**Discussion by anonfawkes73** [utmapp/UTM#4712] [UTM Help (macOS)]
Hey guys, I'm using MacOS Ventura 13.0
VM OS: Ubuntu 22.04.01
Architecture: ARM64 (aarch64)
Machine: OEMU 7.O ARM Virtual Machine (alias of virt-7.0) (virt)

I just had this issue and solved following this: https://github.com/utmapp/UTM/issues/2682#issuecomment-1230327852.
But now, my VM doesn't have access anymore to the network, the connection fails when I try commands such as sudo apt-get update and in the "login page" ipv4 and ipv6 address disappeared.

I have no idea on what happened and how to restore it.
My others VM are still able to connect, as usual.

I'm a beginner user, so if you need any more details please let me know which and how to obtain them.
I attached the config.plist file and I'm available to upload the whole VM (28gb) if needed.
I attached also a screenshot showing the difference between this VM and another with a working connection.

![CleanShot 2022-10-31 at 14 38 05@2x](https://user-images.githubusercontent.com/108414921/199034701-b713e22d-1426-4f68-87c1-ca55f9a345ef.png)

[config.plist.zip](https://github.com/utmapp/UTM/files/9902006/config.plist.zip)

P.S. If not able to fix it, I could use another VM and delete this one, but in order to do so I should copy from the VM to my Mac a file actually stored in this VM, how can I do it since the VM doesn't have a ipv4 address to connect?

**Comment by acrimu** [comment]
After some updates of the Ubuntu in VM, I had the same problem.

Edit 
etc/netplan/00-installer-config.yaml 
to have the correct name for the network interface. Check with ifconfig.
